Dockerfile.template

```
@@ -164,7 +164,7 @@ RUN set -eux; \
 	mv file.c.new file.c; \
 	\
 	autoconf; \
-{{ if is_alpine and ( [ "2.7", "3.0" ] | index(env.version) ) then ( -}}
+{{ if is_alpine and ( [ "2.7", "3.0", "3.1-rc" ] | index(env.version) ) then ( -}}
```

Any idea whether this workaround actually _is_ necessary for 3.1+?  If it is, we should probably update this condition to exclude `2.6` instead, but if not, then we should leave this bit out.

---

I'll see if I can figure it out.

---

I figured you probably don't have access to an s390x machine to test on, so I did a test and 3.1 builds just fine without this block! :smile:

So, I think we should change this back to the following/drop this change (and then run `apply-template.sh` to regenerate appropriately):

```suggestion
{{ if is_alpine and ( [ "2.7", "3.0" ] | index(env.version) ) then ( -}}
```